atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -144,6 +172,20 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji build ID: %s", self.koji_build_id)
 
+        self._get_koji_session()
+
+    def _get_smtp_session(self):
+        try:
+            return get_smtp_session(self.workflow)
+        except KeyError:
+            return smtplib.SMTP(self.smtp_host)
+
+    def _get_koji_session(self):
+        try:
+            self.sesion = get_koji_session(self.workflow)
```

Comment:
Do we need to return after this statement?

Reply:
Also, let's follow the pattern from the other plugins: `_get_koji_session` returns the session object, and caller is the one that assigns to self: `self.session = self._get_koji_session()`.

Reply:
ok will change it so it returns in the same manner as in other plugins